packages/stacked_services/example/lib/ui/views/home_screen.dart

```diff
@@ -92,6 +92,7 @@ class _HomeScreenState extends State<HomeScreen> {
                 await _dialogService.showConfirmationDialog(
                   title: 'Test Confirmation Dialog Title',
                   description: 'Test Confirmation Dialog Description',
+                  barrierDismissible = true,
```

Comment:
This doesn't compile

Reply:
Mixed up the colon notation with equal signs in two places.